experimental/dds/tree2/src/shared-tree-core/branch.ts

```diff
@@ -335,15 +337,12 @@ export class SharedTreeBranch<TEditor extends ChangeFamilyEditor, TChange> exten
 	/**
 	 * Rebase the changes that have been applied to this branch over all the divergent changes in the given branch.
 	 * After this operation completes, this branch will be based off of `branch`.
-	 * @param branch - the head of the branch to rebase onto
-	 * @param undoRedoManager - the undo redo manager of the branch to rebase onto. This is necessary for updating the
-	 * undo redo manager of this branch.
+	 * @param branch - the branch to rebase onto
 	 * @returns the net change to this branch and the commits that were removed and added to this branch by the rebase,
 	 * or undefined if nothing changed
 	 */
 	public rebaseOnto(
-		branch: GraphCommit<TChange>,
-		undoRedoManager: UndoRedoManager<TChange, TEditor>,
+		branch: SharedTreeBranch<TEditor, TChange>,
```

Comment:
Thank you for your service.

Reply:
Haha yea I was pleased that this fell out too :)